CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 All notable changes to this project will be documented in this file.
+## [1.2] - 2019-07-03
+### Changed
+- Change `store_hash_as_json` hash option to default to `true` for regardless of `aggregate_db_storage_type`
+
+## [1.1] - ?
+### Changed
+- Missed
```

Comment:
This is probably my bad

I think this is when I added `aggregate_attribute_changes`...mind updating this?